### ISSUE TITLE
Option to disable incoming email support?

### ISSUE THREAD

**Issue by poVoq** [loomio/loomio#10112]
I noticed that in the latest version it now very prominently displays a "group-name@null" at the top of the group landing page.

I assume this is meant to show the mail-in address?

I don't have this configured at all and would rather like to disable this feature. Previously it was sufficiently hidden that most people would have probably never noticed the lack of support for it.

Thanks for considering this!

**Comment by robguthrie** [comment]
Yes, I figured some people would not want this feature - always good to get feedback. 

I'll add an opt-out ENV option in the next couple of days

**Comment by poVoq** [comment]
Sorry to ask again, but any chance this could be hidden in the next version? Thanks!

**Comment by robguthrie** [comment]
Do you use reply by email?

**Comment by robguthrie** [comment]
I'm just wondering if this should disable the email footer message "you can reply to this email" too. Seems like that would be sensible

**Comment by poVoq** [comment]
No, I don't use it at all, but a small footer somewhere I don't mind as much as the large central UI element that is currently completely broken if you don't setup the reply by email system.

**Comment by Hamsterarsch** [comment]
I could also pick this up, but the scope isn't so clear to me from this conversation.
- Is only hiding the email below the group description enough or is it also important that the backend rejects any incoming mail?
- Should reply by mail be disabled by the same env option (and the footer removed). I ask this because I feel replying by mail is much more intuitive compared to creating a new thread by mail.

**Comment by poVoq** [comment]
From my point of view if would be sufficient to remove showing that functionality in the frontend when it isn't actually available since the incoming email backend isn't running.

Of course removing mentions of it in outgoing email templates would be nice as well, but the main issue is that it is confusing to users to have this big prominent button and pop-up for a feature that doesn't actually work.
